charts/authentik/Chart.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: v2
-description: authentik is an open-source Identity Provider focused on flexibility and versatility
+version: 2023.3.1
```

Comment:
Since this PR includes a breaking change for some users, the major version of the Helm chart should be updated - or at least we need to wait for the next release of Authentik.

Reply:
Yeah, doing major version increments with this versioning scheme is a bit difficult, but it would definitely only be included in the next authentik release